spid_cie_oidc/provider/views.py

```diff
@@ -3,6 +3,7 @@
 import logging
 import urllib.parse
 import uuid
+import json
```

Comment:
where the json library is used?

Reply:
line 438: "attributes": json.dumps(attributes, indent=4)